docker-for-mac/faqs.md

```diff
@@ -112,11 +112,11 @@ For a full explanation and examples, see [I want to connect to a container from
 the Mac](networking.md#i-want-to-connect-to-a-container-from-the-mac) under
 [Known Limitations, Use Cases, and
 Workarounds](networking.md#known-limitations-use-cases-and-workarounds) in the
-Networking topic.
+Networking topic.``
```

Comment:
@mstanleyjones @thaJeztah I have no idea how that happened (stray backticks), nor what it was about :-/ good catch you guys